server/src/db/seeders/ReviewSeeder.js

```diff
@@ -0,0 +1,57 @@
+import { Review } from "../../models/index.js"
+import { Build } from "../../models/index.js"
+import { User } from "../../models/index.js"
+
+class ReviewSeeder {
+    static async seed() {  
+        const build = await Build.query().findOne({title: "Seed Build 1"})
+        const user = await User.query().findOne({email: "test1@email.com"})
```

Comment:
space pad objects
https://github.com/airbnb/javascript?tab=readme-ov-file#whitespace--in-braces